parts/linux/cloud-init/artifacts/ipv6_nftables

```diff
@@ -0,0 +1,12 @@
+define slb_lla = fe80::1234:5678:9abc
+define slb_gua = 2603:1062:0:1:fe80:1234:5678:9abc
+
+add table ip6 azureSLBProbe
+
+flush table ip6 azureSLBProbe
```

Comment:
this wipes the rules in the table before writing them. Only here because we're reconciling this table every hour and for an unknown reason need to reset the rules for the probe packets to flow correctly.